t/op/ref.t

```diff
@@ -265,8 +265,10 @@ is (join('', sort values %$anonhash2), 'BARXYZ');
 # Test bless operator.
 
 package MYHASH;
-
-$object = bless $main'anonhash2;
+{
+    no warnings "syntax", "deprecated";
```

Comment:
Should be `qw(...)` to match the other tests.

Reply:
Fixed.